OpenOversight/tests/conftest.py

```diff
@@ -662,6 +662,8 @@ def teardown():
 
 @pytest.fixture
 def browser(app, request):
+    # FIXME: Skip for now until I can get this working
```

Comment:
Love it! They also had some other stuff in the [previous Dockerfile](https://github.com/lucyparsons/OpenOversight/blob/develop/dockerfiles/web/Dockerfile) that I stripped out because I wanted to slim down the prod image. It may be related too.